### PR TITLE
Python layer update

### PR DESCRIPTION
**Pull request by bhautikpip**
1.  Adds `aiohttp` instrumentation deps to fix downstream CD failures
2. Added instrumentation code from upstream

### CODE REVIEW

File: python/src/otel/otel_sdk/requirements-nodeps.txt [hunk 1]
```diff
@@ -1,4 +1,6 @@
 opentelemetry-instrumentation-aiohttp-client==0.24b0
+opentelemetry-util-http==0.24b0
+asgiref~=3.0
```
Comment (NathanielRN) [inline python/src/otel/otel_sdk/requirements-nodeps.txt:3]: Nit, we should group these 3 dependencies together with a comment. Also we could use lexicographical where possible here (and opentelemetry-util-http==0.24b0 depends on asgiref anyways?).

```suggestion
# All these packages are required for aiohttp instrumentation

asgiref~=3.0
opentelemetry-util-http==0.24b0
opentelemetry-instrumentation-aiohttp-client==0.24b0
```

File: python/src/otel/otel_sdk/otel_wrapper.py [hunk 2]
```diff
@@ -17,32 +18,53 @@
...
-
-def _load_distros():
+def _load_distros() -> BaseDistro:
     for entry_point in iter_entry_points("opentelemetry_distro"):
         try:
-            entry_point.load()().configure()  # type: ignore
-            logger.info("Distribution %s configured", entry_point.name)
+            distro = entry_point.load()()
+            if not isinstance(distro, BaseDistro):
+                logger.debug(
```
Comment (NathanielRN) [inline python/src/otel/otel_sdk/otel_wrapper.py:26]: Should we create a follow up issue to find out why these debug logs can’t be seen in Lambda? But print statements CAN be seen in Lambda?

**Review by NathanielRN — APPROVED** [review]
Great job with this fix 🥳

**Review by wangzlei — APPROVED** [review]

File: python/src/otel/otel_sdk/otel_wrapper.py (hunk 2 above)
Comment (wangzlei) [reply python/src/otel/otel_sdk/otel_wrapper.py:26]: yes please. No idea why the log disappear now.

**Review by wangzlei — APPROVED** [review]

**Pull request merged**